updating/updating-restricted-network-cluster.adoc

```diff
@@ -38,11 +38,31 @@ include::modules/cli-installing-cli.adoc[leveloffset=+3]
 
 // this file doesn't exist, so I'm including the one that should pick up more changes from Clayton's PR - modules/installation-adding-mirror-registry-pull-secret.adoc[leveloffset=+1]
 
-include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+2]
+include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+3]
 
-include::modules/update-mirror-repository.adoc[leveloffset=+2]
+[id="update-mirror-repository_updating-restricted-network-cluster"]
+=== Mirroring the {product-title} image repository
 
-include::modules/machine-health-checks-pausing.adoc[leveloffset=+3]
+Before you update a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+There are two supported methods for mirroring images onto a mirror registry:
+
+* Using the oc-mirror OpenShift CLI (`oc`) plug-in (recommended)
+
+* Using the oc adm release mirror command
+
+Choose one of the supported options below.
```

Comment:
@shellyyang1989 I added the info from lines 48-54 above in response to your feedback, does this look good to you?

Reply:
yes, it looks good to me. Thanks!